<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
    <title><![CDATA[Nick Shek&#039;s Blog]]></title>
    <link href="/b/categories/django.xml" rel="self"/>
    <link href="/"/>
    <updated>2018-10-15T14:56:27+00:00</updated>
    <id>/</id>
        <generator uri="http://sculpin.io/">Sculpin</generator>
            <entry>
            <title type="html"><![CDATA[debug.py =&gt; IndexError: list index out of range]]></title>
            <link href="/b/debug/"/>
            <updated>2015-01-22T00:00:00+00:00</updated>
            <id>/b/debug/</id>
            <content type="html"><![CDATA[<p>若該錯誤來自Django項目的debug.py，解決方法是修改_get_lines_from_file方法，把以下這一行:</p>

<pre><code class="python">source = f.readlines()
</code></pre>

<p>改為</p>

<pre><code class="python">source = f.read().splitlines()
</code></pre>
]]></content>
        </entry>
    </feed>